terraform/gke.tf

```diff
@@ -59,14 +60,15 @@ module "gke" {
     # Add the local IP of the workstation that applies the Terraform to authorized networks
     {
       display_name : "Local IP",
-      cidr_block : "${chomp(data.http.installation_workstation_ip.body)}/32"
+      cidr_block : "${chomp(data.http.installation_workstation_ip.response_body)}/32"
```

Comment:
It's because it has been deprecated in favor of `response_body` (https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http#body)